chapters/descriptorsets.txt

```diff
@@ -2584,6 +2584,10 @@ ifdef::VK_VERSION_1_1,VK_KHR_maintenance1[]
     pname:imageView must: not be 2D or 2D array image view created from a 3D
     image
 endif::VK_VERSION_1_1,VK_KHR_maintenance1[]
+  * If pname:imageView is created from a depth/stencil image the
+    pname:aspectMask used to create the pname:imageView must: include either
+    ename:VK_IMAGE_ASPECT_DEPTH_BIT or ename:VK_IMAGE_ASPECT_STENCIL_BIT but
```

Comment:
Could be simplified as the `VkImageView` already has to be either `VK_IMAGE_ASPECT_DEPTH_BIT`  and\or `VK_IMAGE_ASPECT_STENCIL_BIT`,